contrib/hipew/src/hipew.cpp

```diff
@@ -237,14 +237,14 @@ static void hipewHipExit(void) {
 #ifdef _WIN32
 static int hipewHasOldDriver(const char *hip_path) {
   DWORD verHandle = 0;
-  DWORD verSize = GetFileVersionInfoSize(hip_path, &verHandle);
+  DWORD verSize = GetFileVersionInfoSizeA(hip_path, &verHandle);
```

Comment:
It states in the documentation that :

> The winver.h header defines GetFileVersionInfoSize as an alias which automatically selects the ANSI or Unicode version of this function based on the definition of the UNICODE preprocessor constant.


So perhaps it is not needed ?

Reply:
@KaoCC 
yes the GetFileVersionInfoSize macro auto selects... But in the current design, we are sure the input is char* that's why I simplified the source code.
If the user has configured the VS solution with Advanced -> Character Set = "Use Unicode Character Set" ( which is the default config I think ) then this code won't build because GetFileVersionInfoSize is replaced by GetFileVersionInfoSizeW.

Another solution is to replace the "const char * hip_path"  by  "LPCWSTR hip_path"  ,  and add the "***W("  instead of my "****A(".  ( in case we want this function to handle unicode )...
...or we can also support both hipewHasOldDriverA  and  hipewHasOldDriverW  :)